engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/product-repositories.controller.js

```diff
@@ -49,6 +49,7 @@ angular.module('Bastion.products').controller('ProductRepositoriesController',
         $scope.successMessages = [];
         $scope.errorMessages = [];
 
+        $scope.downloadPolicies = [{id: 'immediate', name: 'Immediate'}, {id: 'on_demand', name: 'On Demand'}, {id: 'background', name: 'Background'}];
```

Comment:
So checksum uses translate (at least for Default). I would imagine that sha256 and sha1 don't need to be translated but I could be wrong.